installer.sh

```diff
@@ -437,6 +435,10 @@ for comp in "agent" "verifier" "tenant" "registrar" "ca" "logging"; do
     fi
 done
 
+# Install templates for configuration upgrades
+mkdir -p /usr/share/keylime/
+cp -r templates /usr/share/keylime/templates
```

Comment:
We can also store them inside the package: https://setuptools.pypa.io/en/stable/userguide/datafiles.html

The advantage of that approach is that it the Python way, the disadvantage is that distributions normally would expect those files in `/usr/share`

Reply:
I'm not a python expert, but I thought the changes I made already included the templates as part of the python distribution package. Please check, and let me know what I need to change to make it to be included.

Reply:
You are right that the files are included. I missed that, sorry. 

Currently we default to use the files `/usr/share/keylime/`, I would instead load them from the package by default (https://docs.python.org/3/library/importlib.resources.html).

Reply:
I will try to use the templates from the package  (when available) as an alternative location when `/usr/share/keylime/templates` is not available. Would this be OK for you?

Reply:
Yes, that sounds like a good idea!

Reply:
So, after reading some documentation, I learned that to use the templates from the package as you suggested
would require changes in many places including changing the whole upgrade process.

The resources are not supposed to be treated as files in the file system, and I use subdirectories to identify the available versions. This means that the logic how the available versions are handled would be different. That would require changes to tests as well.

We would also need to move the `templates` directory to be inside the keylime package (i.e.`keylime/templates`), as well as installing them as part of the binary distribution. The templates would have to become a subpackage (i.e. `import keylime.templates` works).

For me, this looks like a lot of work that would complicate the code and maintainability (as we would have two separate ways of handling the files and detecting the available versions to be tested and maintained), with little gain.

If this is really required, I would work on it on a separate PR.

Reply:
Yeah, that sounds more complicated than I thought. Putting those files into `/usr/share` is standard for Linux packaging, so I'm fine with using that!